web/survey/utils/constants.py

```diff
@@ -0,0 +1,2 @@
+"""Constants for survey run naming patterns and matching survey components."""
+RUN_NAME_TO_SURVEY_COMPONENT = {r"^SB.*_gc$": "Quality-Check", r"^SER.*$": "WALLABY v1"}
```

Comment:
`^SB.*_gc$` probably should be `^SB.*_qc$` (quality check)

Reply:
Nicely spotted!